### PR TITLE
Version handling

### PR DESCRIPTION
**Pull request by sbellem**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -1,5 +1,15 @@
+import os
+import sys
+
 import sphinx_rtd_theme
 
+cwd = os.getcwd()
+project_root = os.path.dirname(cwd)
+sys.path.insert(0, project_root)
+
+import cryptoconditions     # noqa
```
Comment (r-marques) [inline docs/conf.py]: is it really necessary to import cryptoconditions here?
This means that cryptoconditions needs to be installed in order to build the docs

Comment (sbellem) [inline docs/conf.py]: Good question. I am not sure.

The [`cookiecutter`](https://github.com/audreyr/cookiecutter) project does it this way. We could also read the file like it's being done for `bigchaindb` right now.

Regarding the need to install `cryptoconditions` itself, `readthedocs` installs a project via `python setup.py install` before building the docs.

As soon as we do slightly sophisticated things like using `doctests` or [`autorun`](https://bitbucket.org/birkenfeld/sphinx-contrib/src/55b13a74d56eebdbf3b6c11b1e73d2718ea51ab7/autorun/?at=default) (to automatically generate the output), we'll need to install the project itself.

So I wonder whether the requirement to install `cryptoconditions` is a problem in this case.

But, anyhow, I am open to change it to simply parsing `version.py`.

**Comment by ttmc** [comment]
Codecov is complaining about `version.py`. You could tell it to ignore that file by adding a `codecov.yml` file to the repo root directory, with instructions to ignore `version.py`. You could use the `codecov.yml` file in the bigchaindb/bigchaindb repo as a starting point:

https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/codecov.yml

File: docs/conf.py (hunk 1 above)
Comment (r-marques) [inline docs/conf.py]: I'm not sure.

If you do it the same way you do it in `setup.py` at least we will be consistent

Comment (sbellem) [inline docs/conf.py]: @r-marques  ok, done the change

as a side note, the disadvantage of reading the file is that its path is hardcoded, and if somehow the relative location of `conf.py` would change, with respect to `version.py`, we would need to update `conf.py` -- but that is really a minor detail at this point

**Comment by r-marques** [comment]
:+1:

**Pull request merged**